source/sass/blocks/gift.scss

```
@@ -0,0 +1,45 @@
+.gift {
+  padding: 0 0 0 0;
+  box-sizing: border-box;
+  position: relative;
+  z-index: 1;
+  display: block;
+  min-height: 289px;
+
+  &::before {
+    content: "";
+    position: absolute;
+    min-height: 289px;
+    width: 100%;
+    background-image: url("../img/background/gift-background-mobile.jpg");
```

```suggestion
    background-image: url("../../img/background/gift-background-mobile.jpg");
```

нужно поправить все урлы, не только фонты ушли вглубь же